src/main/webapp/src/components/RatingList.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import { ListItemIcon } from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginTop: "50px",
+    width: "100%",
+    display: "flex",
+    alignItems: "center",
+    flexDirection: "column",
+  },
+  listIcon: {
+    fontFamily: "ABeeZee",
+    fontSize: "28px",
+    color: "#3A3838",
+  },
+}));
+
+function EnumeratedList(props) {
+  const listItems = props.listItems;
+  const classes = props.styleClasses;
+  const reactListItems = listItems.map((item, index) => (
+    <ListItem key={index + 1}>
+      <ListItemIcon className={classes.listIcon}>
+        {index + 1}
+      </ListItemIcon>
+      <ListItemText
+        primary={item.songName}
+        secondary={item.bandName + '. From album "' + item.albumName + '"'}
+      />
+    </ListItem>
+  ));
+  return <List>{reactListItems}</List>;
+}
+
+export default function RatingList() {
+  const classes = useStyles();
+
```

Comment:
I think you should leave a comment telling people we will add songs automatically later.
Something like
// TODO Fetch top songs from database.